_notices/rsn0018.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 18 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Support for Rocky 8.5 in v22.08"
+notice_author: RAPIDS Ops
+notice_status: In Progress
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v22.08+"
+notice_created: 2022-06-08
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2022-06-08
+---
+
+## Overview
+
+Development effort will begin supporting `Rocky 8.5` in `v22.08` nightlies. `Rocky 8.5` will replace the `CentOS 8` OS distributions for future RAPIDS releases.  Support will be experimental for `v22.08`. GA support for `Rocky 8.5` is expected in `v22.08`
```

Comment:
```suggestion
Development effort will begin supporting `Rocky 8.6` in `v22.08` nightlies. `Rocky 8.6` will replace the `CentOS 8` OS distributions for future RAPIDS releases.  Support will be experimental for `v22.08`. GA support for `Rocky 8.5` is expected in `v22.10`
```

Reply:
I updated this to be 22.08 nighties get EA, vs stable getting GA.  Will we instead have GA for 22.10 stable and not 22.08 stable? @aravenel

Reply:
read @raydouglass 's comment, okay, i'll follow that timeline instead